From fd1432712d30fa258b8a1118c87e6989b8af5571 Mon Sep 17 00:00:00 2001
From: Jean-Yves Avenard <jyavenard@mythtv.org>
Date: Mon, 30 Jun 2014 14:16:12 -0700
Subject: [PATCH 10/12] Increase size of streaming buffer

streaming buffer size is also used to detect audio format, 16kB may not be sufficient to properly detect corrupted media or FLAC ones.
The buffer will be resized to 32kB by FFmpeg once the file has been successfully opened
---
 mythplugins/mythmusic/mythmusic/avfdecoder.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mythplugins/mythmusic/mythmusic/avfdecoder.cpp b/mythplugins/mythmusic/mythmusic/avfdecoder.cpp
index d619a92..589f715 100644
--- a/mythplugins/mythmusic/mythmusic/avfdecoder.cpp
+++ b/mythplugins/mythmusic/mythmusic/avfdecoder.cpp
@@ -49,7 +49,7 @@ extern "C" {
 }
 
 // size of the buffer used for streaming
-#define BUFFER_SIZE 16384
+#define BUFFER_SIZE 65536
 
 // streaming callbacks
 static int ReadFunc(void *opaque, uint8_t *buf, int buf_size)
-- 
1.7.10.2

